spotty/spotty/Utils/Extensions/UI/UIView+Animations.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  UIView+Animations.swift
+//  spotty
+//
+//  Created by Ivaylo Kalaydzhiev on 9.06.22.
+//
+
+import UIKit
+
+extension UIView {
+    
+    /// The UI View animates a spring bounce
+    func bounce() {
```

Comment:
I suggest to add parameters here for duration, delay, ..... and add default values to them in order animation to be more configurable.

Reply:
done